rotkehlchen/db/upgrades/v40_v41.py

```diff
@@ -22,15 +26,35 @@ def _add_cache_table(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_cache_table')
 
 
+def _move_non_settings_mappings_to_cache(write_cursor: 'DBCursor') -> None:
+    """Move the non-settings value from `settings` to a seperate `key_value_cache` table"""
+    log.debug('Enter _move_non_settings_mappings_to_cache')
+    movable_settings = write_cursor.execute(
+        f'SELECT name, value FROM settings WHERE name IN ({",".join(["?"] * len(SETTINGS_MOVED))});',  # noqa: E501
+        tuple(SETTINGS_MOVED),
+    ).fetchall()
```

Comment:
if it is used in one here don't create a global variable

```suggestion
    settings_moved = {x.serialize() for x in DBCache}
    movable_settings = write_cursor.execute(
        f'SELECT name, value FROM settings WHERE name IN ({",".join(["?"] * len(settings_moved))});',  # noqa: E501
        tuple(settings_moved),
    ).fetchall()
```

all of them have the same name right?